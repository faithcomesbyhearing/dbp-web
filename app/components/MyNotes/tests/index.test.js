import React from 'react';
import { render } from '@testing-library/react';
import MyNotes from '..';
import {
	highlights,
	bookmarks,
	vernacularNamesObject,
	notes,
} from '../../../utils/testUtils/notebookFunctions';

const props = {
	getNotes: jest.fn(),
	deleteNote: jest.fn(),
	setPageSize: jest.fn(),
	getBookmarks: jest.fn(),
	getHighlights: jest.fn(),
	setActiveNote: jest.fn(),
	setActivePage: jest.fn(),
	setActiveChild: jest.fn(),
	deleteBookmark: jest.fn(),
	updateHighlight: jest.fn(),
	deleteHighlights: jest.fn(),
	toggleNotesModal: jest.fn(),
	togglePageSelector: jest.fn(),
	listData: notes,
	bookmarkList: bookmarks,
	highlights,
	vernacularNamesObject,
	sectionType: 'notes',
	pageSize: 10,
	totalPages: 1,
	activePage: 1,
	pageSizeBookmark: 10,
	totalPagesBookmark: 1,
	activePageBookmark: 1,
	pageSizeHighlight: 10,
	totalPagesHighlight: 1,
	activePageHighlight: 1,
	pageSelectorState: false,
};

describe('<MyNotes /> container', () => {
	it('should match snapshot with default props', () => {
		const { asFragment } = render(<MyNotes {...props} />);
		expect(asFragment()).toMatchSnapshot();
	});
	it('should match snapshot with highlights active', () => {
		const { asFragment } = render(
			<MyNotes {...props} sectionType={'highlights'} />,
		);
		expect(asFragment()).toMatchSnapshot();
	});
	it('should match snapshot with bookmarks active', () => {
		const { asFragment } = render(
			<MyNotes {...props} sectionType={'bookmarks'} />,
		);
		expect(asFragment()).toMatchSnapshot();
	});
	it('should match snapshot with page selector open', () => {
		const { asFragment } = render(<MyNotes {...props} pageSelectorState />);
		expect(asFragment()).toMatchSnapshot();
	});
});
