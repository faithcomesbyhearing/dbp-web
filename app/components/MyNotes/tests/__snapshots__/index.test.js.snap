// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyNotes /> container should match snapshot with bookmarks active 1`] = `
<DocumentFragment>
  <div
    class="list-sections"
  >
    <div
      class="searchbar"
    >
      <span
        class="input-wrapper"
      >
        <svg
          class="icon"
        >
          <use
            xlink:href="/svglist.svg#search"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          />
        </svg>
        <input
          placeholder="SEARCH BOOKMARKS"
          value=""
        />
      </span>
    </div>
    <section
      class="note-list"
    >
      <div
        class="highlight-item"
        id="5452296"
      >
        <a
          href="/bible/ENGESV/MAT/1"
        >
          <button
            class="list-item"
            type="button"
          >
            <div
              class="title-text"
            >
              <h4
                class="title"
              >
                <span
                  class="date"
                >
                  01.18.19
                </span>
                 | Matthew 1:2
              </h4>
              <p
                class="text"
              >
                ENGESV
              </p>
            </div>
          </button>
        </a>
        <button
          class="delete-highlight"
          tabindex="0"
          type="button"
        >
          <svg
            class="icon"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </button>
      </div>
      <div
        class="highlight-item"
        id="5452373"
      >
        <a
          href="/bible/ENGESV/MAT/1"
        >
          <button
            class="list-item"
            type="button"
          >
            <div
              class="title-text"
            >
              <h4
                class="title"
              >
                <span
                  class="date"
                >
                  01.18.19
                </span>
                 | Matthew 1:2
              </h4>
              <p
                class="text"
              >
                ENGESV
              </p>
            </div>
          </button>
        </a>
        <button
          class="delete-highlight"
          tabindex="0"
          type="button"
        >
          <svg
            class="icon"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </button>
      </div>
      <div
        class="highlight-item"
        id="5574789"
      >
        <a
          href="/bible/ENGESV/MRK/16"
        >
          <button
            class="list-item"
            type="button"
          >
            <div
              class="title-text"
            >
              <h4
                class="title"
              >
                <span
                  class="date"
                >
                  02.08.19
                </span>
                 | Mark 16:3
              </h4>
              <p
                class="text"
              >
                ENGESV
              </p>
            </div>
          </button>
        </a>
        <button
          class="delete-highlight"
          tabindex="0"
          type="button"
        >
          <svg
            class="icon"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </button>
      </div>
      <div
        class="highlight-item"
        id="5574790"
      >
        <a
          href="/bible/ENGNIV/MAT/11"
        >
          <button
            class="list-item"
            type="button"
          >
            <div
              class="title-text"
            >
              <h4
                class="title"
              >
                <span
                  class="date"
                >
                  02.12.19
                </span>
                 | Matthew 11:7
              </h4>
              <p
                class="text"
              >
                ENGNIV
              </p>
            </div>
          </button>
        </a>
        <button
          class="delete-highlight"
          tabindex="0"
          type="button"
        >
          <svg
            class="icon"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </button>
      </div>
      <div
        class="highlight-item"
        id="5574788"
      >
        <a
          href="/bible/ENGESV/MAT/3"
        >
          <button
            class="list-item"
            type="button"
          >
            <div
              class="title-text"
            >
              <h4
                class="title"
              >
                <span
                  class="date"
                >
                  02.06.19
                </span>
                 | Matthew 3:3
              </h4>
              <p
                class="text"
              >
                ENGESV
              </p>
            </div>
          </button>
        </a>
        <button
          class="delete-highlight"
          tabindex="0"
          type="button"
        >
          <svg
            class="icon"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </button>
      </div>
    </section>
    <div
      class="pagination"
    >
      <div
        class="item-list"
      >
        <div
          class="item active"
        >
          <span
            id="1-button"
            role="button"
            tabindex="0"
          >
            1
          </span>
        </div>
      </div>
      <div
        class="page-size-selector"
      >
        <div
          class="closed"
        >
          <div
            class="active"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              10 PER PAGE
            </span>
            <svg
              height="13px"
              width="7px"
            >
              <use
                xlink:href="/svglist.svg#pagination-arrows"
                xmlns:xlink="http://www.w3.org/1999/xlink"
              />
            </svg>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              25 PER PAGE
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              50 PER PAGE
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              100 PER PAGE
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              VIEW ALL
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
        </div>
        <div
          class="per-page-toggle"
          role="button"
          tabindex="0"
        >
          10 PER PAGE
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`<MyNotes /> container should match snapshot with default props 1`] = `
<DocumentFragment>
  <div
    class="list-sections"
  >
    <div
      class="searchbar"
    >
      <span
        class="input-wrapper"
      >
        <svg
          class="icon"
        >
          <use
            xlink:href="/svglist.svg#search"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          />
        </svg>
        <input
          placeholder="SEARCH NOTES"
          value=""
        />
      </span>
    </div>
    <section
      class="note-list"
    >
      <div
        class="highlight-item"
        id="403619"
      >
        <div
          class="list-item"
          role="button"
          tabindex="0"
        >
          <div
            class="title-text"
          >
            <h4
              class="title"
            >
              <span
                class="date"
              >
                02.12.19
              </span>
               | new note
            </h4>
            <p
              class="text"
            >
              newest note
            </p>
          </div>
        </div>
        <div
          class="edit-note"
          id="403619-edit"
          role="button"
          tabindex="0"
        >
          <svg
            class="icon"
          >
            <use
              xlink:href="/svglist.svg#edit_note"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Edit
          </span>
        </div>
        <div
          class="delete-highlight"
          id="403619-delete"
          role="button"
          tabindex="0"
        >
          <svg
            class="icon"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </div>
      </div>
      <div
        class="highlight-item"
        id="403618"
      >
        <div
          class="list-item"
          role="button"
          tabindex="0"
        >
          <div
            class="title-text"
          >
            <h4
              class="title"
            >
              <span
                class="date"
              >
                02.06.19
              </span>
               | New Note
            </h4>
            <p
              class="text"
            >
              Is here
            </p>
          </div>
        </div>
        <div
          class="edit-note"
          id="403618-edit"
          role="button"
          tabindex="0"
        >
          <svg
            class="icon"
          >
            <use
              xlink:href="/svglist.svg#edit_note"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Edit
          </span>
        </div>
        <div
          class="delete-highlight"
          id="403618-delete"
          role="button"
          tabindex="0"
        >
          <svg
            class="icon"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </div>
      </div>
    </section>
    <div
      class="pagination"
    >
      <div
        class="item-list"
      >
        <div
          class="item active"
        >
          <span
            id="1-button"
            role="button"
            tabindex="0"
          >
            1
          </span>
        </div>
      </div>
      <div
        class="page-size-selector"
      >
        <div
          class="closed"
        >
          <div
            class="active"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              10 PER PAGE
            </span>
            <svg
              height="13px"
              width="7px"
            >
              <use
                xlink:href="/svglist.svg#pagination-arrows"
                xmlns:xlink="http://www.w3.org/1999/xlink"
              />
            </svg>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              25 PER PAGE
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              50 PER PAGE
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              100 PER PAGE
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              VIEW ALL
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
        </div>
        <div
          class="per-page-toggle"
          role="button"
          tabindex="0"
        >
          10 PER PAGE
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`<MyNotes /> container should match snapshot with highlights active 1`] = `
<DocumentFragment>
  <div
    class="list-sections"
  >
    <div
      class="searchbar"
    >
      <span
        class="input-wrapper"
      >
        <svg
          class="icon"
        >
          <use
            xlink:href="/svglist.svg#search"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          />
        </svg>
        <input
          placeholder="SEARCH HIGHLIGHTS"
          value=""
        />
      </span>
    </div>
    <section
      class="note-list"
    >
      <div
        class="highlight-item"
        id="24440733_rgba(173,221,121,0.7)"
      >
        <a
          href="/bible/ENGNAB/EPH/4"
        >
          <button
            class="list-item"
            type="button"
          >
            <div
              class="title-text"
            >
              <h4
                class="title"
              >
                EPH - 4:14 - (ENGNAB)
              </h4>
              <h4
                class="text"
              >
                ENGNAB
              </h4>
            </div>
          </button>
        </a>
        <button
          class="edit-color"
          id="24440733_rgba(173,221,121,0.7)-edit"
          tabindex="0"
          type="button"
        >
          <span
            class="color-icon"
            style="background-color: rgba(173, 221, 121, 0.7);"
          />
          <span>
            Edit
          </span>
        </button>
        <div
          class="delete-highlight"
        >
          <svg
            class="icon"
            id="24440733_rgba(173,221,121,0.7)-delete"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </div>
      </div>
      <div
        class="highlight-item"
        id="25827775_rgba(252,230,0,0.5)"
      >
        <a
          href="/bible/ENGESV/MAT/3"
        >
          <button
            class="list-item"
            type="button"
          >
            <div
              class="title-text"
            >
              <h4
                class="title"
              >
                MAT - 3:3 - (ENGESV)
              </h4>
              <h4
                class="text"
              >
                ENGESV
              </h4>
            </div>
          </button>
        </a>
        <button
          class="edit-color"
          id="25827775_rgba(252,230,0,0.5)-edit"
          tabindex="0"
          type="button"
        >
          <span
            class="color-icon"
            style="background-color: rgba(252, 230, 0, 0.5);"
          />
          <span>
            Edit
          </span>
        </button>
        <div
          class="delete-highlight"
        >
          <svg
            class="icon"
            id="25827775_rgba(252,230,0,0.5)-delete"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </div>
      </div>
      <div
        class="highlight-item"
        id="25827777_rgba(84,185,72,0.5)"
      >
        <a
          href="/bible/ENGESV/MAT/1"
        >
          <button
            class="list-item"
            type="button"
          >
            <div
              class="title-text"
            >
              <h4
                class="title"
              >
                MAT - 1:4 - (ENGESV)
              </h4>
              <h4
                class="text"
              >
                ENGESV
              </h4>
            </div>
          </button>
        </a>
        <button
          class="edit-color"
          id="25827777_rgba(84,185,72,0.5)-edit"
          tabindex="0"
          type="button"
        >
          <span
            class="color-icon"
            style="background-color: rgba(84, 185, 72, 0.5);"
          />
          <span>
            Edit
          </span>
        </button>
        <div
          class="delete-highlight"
        >
          <svg
            class="icon"
            id="25827777_rgba(84,185,72,0.5)-delete"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </div>
      </div>
      <div
        class="highlight-item"
        id="25827778_rgba(80,165,220,0.5)"
      >
        <a
          href="/bible/ENGESV/MAT/1"
        >
          <button
            class="list-item"
            type="button"
          >
            <div
              class="title-text"
            >
              <h4
                class="title"
              >
                MAT - 1:17 - (ENGESV)
              </h4>
              <h4
                class="text"
              >
                ENGESV
              </h4>
            </div>
          </button>
        </a>
        <button
          class="edit-color"
          id="25827778_rgba(80,165,220,0.5)-edit"
          tabindex="0"
          type="button"
        >
          <span
            class="color-icon"
            style="background-color: rgba(80, 165, 220, 0.5);"
          />
          <span>
            Edit
          </span>
        </button>
        <div
          class="delete-highlight"
        >
          <svg
            class="icon"
            id="25827778_rgba(80,165,220,0.5)-delete"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </div>
      </div>
      <div
        class="highlight-item"
        id="25827779_rgba(252,230,0,0.5)"
      >
        <a
          href="/bible/ENGESV/MAT/1"
        >
          <button
            class="list-item"
            type="button"
          >
            <div
              class="title-text"
            >
              <h4
                class="title"
              >
                MAT - 1:6 - (ENGESV)
              </h4>
              <h4
                class="text"
              >
                ENGESV
              </h4>
            </div>
          </button>
        </a>
        <button
          class="edit-color"
          id="25827779_rgba(252,230,0,0.5)-edit"
          tabindex="0"
          type="button"
        >
          <span
            class="color-icon"
            style="background-color: rgba(252, 230, 0, 0.5);"
          />
          <span>
            Edit
          </span>
        </button>
        <div
          class="delete-highlight"
        >
          <svg
            class="icon"
            id="25827779_rgba(252,230,0,0.5)-delete"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </div>
      </div>
      <div
        class="highlight-item"
        id="25827776_rgba(84,185,72,0.5)"
      >
        <a
          href="/bible/ENGESV/MRK/16"
        >
          <button
            class="list-item"
            type="button"
          >
            <div
              class="title-text"
            >
              <h4
                class="title"
              >
                MRK - 16:5 - (ENGESV)
              </h4>
              <h4
                class="text"
              >
                ENGESV
              </h4>
            </div>
          </button>
        </a>
        <button
          class="edit-color"
          id="25827776_rgba(84,185,72,0.5)-edit"
          tabindex="0"
          type="button"
        >
          <span
            class="color-icon"
            style="background-color: rgba(84, 185, 72, 0.5);"
          />
          <span>
            Edit
          </span>
        </button>
        <div
          class="delete-highlight"
        >
          <svg
            class="icon"
            id="25827776_rgba(84,185,72,0.5)-delete"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </div>
      </div>
    </section>
    <div
      class="pagination"
    >
      <div
        class="item-list"
      >
        <div
          class="item active"
        >
          <span
            id="1-button"
            role="button"
            tabindex="0"
          >
            1
          </span>
        </div>
      </div>
      <div
        class="page-size-selector"
      >
        <div
          class="closed"
        >
          <div
            class="active"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              10 PER PAGE
            </span>
            <svg
              height="13px"
              width="7px"
            >
              <use
                xlink:href="/svglist.svg#pagination-arrows"
                xmlns:xlink="http://www.w3.org/1999/xlink"
              />
            </svg>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              25 PER PAGE
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              50 PER PAGE
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              100 PER PAGE
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              VIEW ALL
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
        </div>
        <div
          class="per-page-toggle"
          role="button"
          tabindex="0"
        >
          10 PER PAGE
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`<MyNotes /> container should match snapshot with page selector open 1`] = `
<DocumentFragment>
  <div
    class="list-sections"
  >
    <div
      class="searchbar"
    >
      <span
        class="input-wrapper"
      >
        <svg
          class="icon"
        >
          <use
            xlink:href="/svglist.svg#search"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          />
        </svg>
        <input
          placeholder="SEARCH NOTES"
          value=""
        />
      </span>
    </div>
    <section
      class="note-list"
    >
      <div
        class="highlight-item"
        id="403619"
      >
        <div
          class="list-item"
          role="button"
          tabindex="0"
        >
          <div
            class="title-text"
          >
            <h4
              class="title"
            >
              <span
                class="date"
              >
                02.12.19
              </span>
               | new note
            </h4>
            <p
              class="text"
            >
              newest note
            </p>
          </div>
        </div>
        <div
          class="edit-note"
          id="403619-edit"
          role="button"
          tabindex="0"
        >
          <svg
            class="icon"
          >
            <use
              xlink:href="/svglist.svg#edit_note"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Edit
          </span>
        </div>
        <div
          class="delete-highlight"
          id="403619-delete"
          role="button"
          tabindex="0"
        >
          <svg
            class="icon"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </div>
      </div>
      <div
        class="highlight-item"
        id="403618"
      >
        <div
          class="list-item"
          role="button"
          tabindex="0"
        >
          <div
            class="title-text"
          >
            <h4
              class="title"
            >
              <span
                class="date"
              >
                02.06.19
              </span>
               | New Note
            </h4>
            <p
              class="text"
            >
              Is here
            </p>
          </div>
        </div>
        <div
          class="edit-note"
          id="403618-edit"
          role="button"
          tabindex="0"
        >
          <svg
            class="icon"
          >
            <use
              xlink:href="/svglist.svg#edit_note"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Edit
          </span>
        </div>
        <div
          class="delete-highlight"
          id="403618-delete"
          role="button"
          tabindex="0"
        >
          <svg
            class="icon"
          >
            <use
              xlink:href="/svglist.svg#delete"
              xmlns:xlink="http://www.w3.org/1999/xlink"
            />
          </svg>
          <span>
            Delete
          </span>
        </div>
      </div>
    </section>
    <div
      class="pagination"
    >
      <div
        class="item-list"
      >
        <div
          class="item active"
        >
          <span
            id="1-button"
            role="button"
            tabindex="0"
          >
            1
          </span>
        </div>
      </div>
      <div
        class="page-size-selector"
      >
        <div
          class="open"
        >
          <div
            class="active"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              10 PER PAGE
            </span>
            <svg
              height="13px"
              width="7px"
            >
              <use
                xlink:href="/svglist.svg#pagination-arrows"
                xmlns:xlink="http://www.w3.org/1999/xlink"
              />
            </svg>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              25 PER PAGE
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              50 PER PAGE
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              100 PER PAGE
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
          <div
            class="inactive"
            role="button"
            tabindex="0"
          >
            <span
              class="text"
            >
              VIEW ALL
            </span>
            <span
              class="svg-placeholder"
            >
               
            </span>
          </div>
        </div>
        <div
          class="closed per-page-toggle"
          role="button"
          tabindex="0"
        >
          10 PER PAGE
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
